### ISSUE TITLE
Summary tables should be in repo.

### ISSUE THREAD

**Issue by petermr** [rossmounce/pluto-ONS#2]
Intermediate summary files should be posted to repo.

**Comment by rossmounce** [comment]
It's in there. There's just too many files in the folder for it to be displayed visually on github. Perhaps I should put reports in a different folder? https://github.com/rossmounce/pluto-ONS/blob/master/testing/batch-20150821-output/all-input/summarytable.txt

**Comment by petermr** [comment]
Good idea. That's one reason why batches are a good idea 20 batches of 250 in 20 subdirectories works well. You can browse them.

(I don't think people, including me, will download 5000 files to see what's in the repo :-).
